charts/opentelemetry-collector/examples/deployment-otlp-traces/values.yaml

```diff
@@ -1,6 +1,9 @@
 mode: deployment
 
 ports:
+  otlp:
+    enabled: true
+    nodePort: 30317
```

Comment:
@mllab-nl we use the rendered examples to help users with common situations or complicated configuration.  I don't think `nodePort` is going to be common nor is it a complicated field to use when needed so I don't think we need to update any examples.

For your local testing you should create a values.yaml with all the fields set that will test `nodePort` (you can change service type with `service.type` in the values.yaml)

Reply:
Removed. 
Local check confirm that with following config I get proper service with correct ports deployed:

> 
> 
> mode: deployment
> 
> ports:
>   otlp:
>     nodePort: 30117
>   otlp-http:    
>     nodePort: 30118
>   jaeger-compact:
>     enabled: false
>   jaeger-thrift:
>     enabled: false
>   jaeger-grpc:
>     enabled: false
>   zipkin:
>     enabled: false
> 
> service:
>   type: NodePort